plugins/generator-1.16.5/forge-1.16.5/templates/block.java.ftl

```diff
@@ -308,7 +308,8 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 				list.add(new StringTextComponent(<@procedureOBJToTextCode data.specialInformation/>));
 			<#else>
 				<#list data.specialInformation as entry>

```

Comment:
This is how it should look like:
```freemarker
				<#list thelper.splitCommaSeparatedStringListWithEscapes(data.specialInformation) as entry>
				list.add(new StringTextComponent("${JavaConventions.escapeStringForJava(entry)}"));
				</#list>
```
Same thing about 1.18.2.

Reply:
Oh ok thx